app/Main.hs

```diff
@@ -88,7 +89,16 @@ main = customExecParser (prefs showHelpOnError) opts >>= \case
     let handleReverse = bool id reverseEdges optReverse
     graph <- handleReverse <$> getInput optGraph
     case optCommand of
-        Why from to      -> printStrs $ [from] <> ((" imports " <>) <$> why graph from to)
+        Why{..}      ->
+          if showAll
+          then
+              drawTree
+            $ fmap T.unpack
+            $ graphToTree fromModule
+            $ allPathsTo graph fromModule toModule
+          else
+            let explainer = if optReverse then " imports " else " imported by "
```

Comment:
This was backwards:

```
$ graphex graph -g graph.json why Graphex.Core Graphex
Graphex.Core
 imports Graphex
```

I fixed it to be correct and take `-r` into account:

```
$ graphex graph -g graph.json why Graphex.Core Graphex
Graphex.Core
 imported by Graphex

$ graphex graph -r -g graph.json why Graphex Graphex.Core Graphex  imports Graphex.Core
```

Reply:
Yeah, I was noticing that.  Thanks for clearing it up.